app-editors/bluefish/bluefish-2.2.12.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit autotools gnome2-utils python-single-r1 xdg-utils
```

Comment:
Looks to me like you don't need *gnome2-utils.eclass* here, the functionality what you use has been moved to *xdg-utils.eclass* with EAPI-7. Although I'd suggest you to just inherit *xdg.eclass* here.